Starbuck

fsansalvadore opened the pull request.

fsansalvadore committed:
d4 setup

fsansalvadore committed:
auth title

fsansalvadore committed:
Starbuck

fsansalvadore committed:
update video id

Commit:
starbuck main stage (#19725)

* Caption fixes for LWXd3 (#19680)

* Update 2023-12-13-supabase-branching.mdx

* Update 2023-12-13-supabase-branching.mdx

* fix: add requirement on Docker client for edge funcs deploy (#19624)

* fix: add requirement on Docker client for edge funcs deploy

* fix: docker name

Co-authored-by: Han Qiao <sweatybridge@gmail.com>

* Update apps/docs/pages/guides/functions/deploy.mdx

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

* fix: prettier

---------

Co-authored-by: Han Qiao <sweatybridge@gmail.com>
Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

* Add YouTube video, form link, more internal linking. (#19685)

* Add YouTube video, form link, more internal linking.

* More internal linking.

* Edit admonition.

* Swap OG image, add planetpg tag

* Prettier

* Update name

---------

Co-authored-by: Terry Sutton <saltcod@gmail.com>

* fix: update basic auth settings form (#19708)

* fix: allow formfield to not have an icon

* fix: convert radio buttons to formfields

* fix: update copy & icons in auth settings page (#19710)

* fix: add hcaptcha and cloudflare icons

* fix: update copy

* Chore/deprecate publications store (#19655)

* Hide RLS assistant features for orgs with HIPAA

* Replace all usage of meta.publications with react query

* Deprecate publications store from meta store mobx, shift updateTableRealtime logic to SidePanelEditor

* Address comments

* Fix create trigger (#19711)

* Chore/deprecate triggers store (#19712)

* Replace all usage of meta.triggers with react query

* Deprecate and remove ttriggers mobx store

* Fix

* Update apps/studio/components/interfaces/Database/Triggers/TriggersList/TriggersList.tsx

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

---------

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

* Chore/branching update requirements (#19670)

* Update requirements to enable branching

* Remove panel query when closing subscripton plan panel

* Update apps/studio/components/layouts/AppLayout/EnableBranchingButton/BranchingPlanNotice.tsx

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

* Lint

* Lint

* Update

---------

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

* Update branching info (#19715)

* Update branching info

* swap

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

* fix: time until hackathon end count down clock (#19716)

* build-d4 (#19709)

* build-d4

* Add images and more links.

* Update description.

* ADd mdx to filename.

* Fix and update images

* Edit filename.

* replace notion links and prettify doc

* fix title image link

* update title

* Update apps/www/_blog/2023-12-14-supabase-wrappers-v02-now-available.mdx

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* Update apps/www/_blog/2023-12-14-supabase-wrappers-v02-now-available.mdx

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* update filename,

---------

Co-authored-by: Bo Lu <lv.patrick@gmail.com>
Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* docs: auth passwords page (#19490)

* docs: auth passwords page

* Add redirect from old url

* apply suggestion from @charislam

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

---------

Co-authored-by: Kamil Ogórek <kamil.ogorek@gmail.com>
Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>
Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Chore/deprecate functions store (#19714)

* Shift all usage of database functions from pg meta store to RQ

* Remove functions store from pg meta mobx store

* fix

* Fix

* Fixes

* fix

* Address comments

* Add build d4 (#19721)

add build item

* chore: add support sla of plans and non abuse copy (#19572)

* chore: add support sla of plans and non abuse copy
reword the SLA description of the popup to cover all paid plans
add support sla shown in support form based on plan
default expand support sla shown in support form
add non-abuse copy if user tries to submit high or urgent severity tickets

* fix: mispelling on prioritized

* fix: update copy based on comments

* starbucks-main-stage

* Update file name.

* Add og and thumb.

* Add video.

* Add LWX links.

* chore: Migrate profile information to use `react-hook-form` (#19675)

* Fix the icon in the theme listbox.

* Remove visible prop on FormSection since it used block which clashes with grid CSS class.

* Migrate the user profile form to use react-hook-form.

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* chore: change current to all local for migrations in db reset command (#19728)

* change current to all local for migrations in db reset command

* Apply suggestions from code review

---------

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* chore: Fix the types in the `ui` package (#19633)

* Enable the typecheck command for the ui package.

* Ignore the stories.tsx files when typechecking.

* Remove the unused Avatar component. We can use the one provided by shadcn now.

* Rename the datatable.stories file.

* Fix various type errors.

* Fix the Toast component.

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Update apps/www/_blog/2023-12-14-supabase-auth-dentity-linking-session-control.mdx

Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Add images.

* Fix description.

* Add new date and admonition.

* Fix URL.

* Update apps/www/_blog/2023-12-14-supabase-auth-identity-linking-hooks.mdx

---------

Co-authored-by: Jonathan Summers-Muir <MildTomato@users.noreply.github.com>
Co-authored-by: Lakshan Perera <lakshan@supabase.io>
Co-authored-by: Han Qiao <sweatybridge@gmail.com>
Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>
Co-authored-by: Terry Sutton <saltcod@gmail.com>
Co-authored-by: Kang Ming <kang.ming1996@gmail.com>
Co-authored-by: Joshen Lim <joshenlimek@gmail.com>
Co-authored-by: Alaister Young <alaister@users.noreply.github.com>
Co-authored-by: Tyler <18113850+dshukertjr@users.noreply.github.com>
Co-authored-by: Bo Lu <lv.patrick@gmail.com>
Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>
Co-authored-by: Stojan Dimitrovski <sdimitrovski@gmail.com>
Co-authored-by: Kamil Ogórek <kamil.ogorek@gmail.com>
Co-authored-by: Francesco Sansalvadore <f.sansalvadore@gmail.com>
Co-authored-by: Pamela Chia <pamelachiamayyee@gmail.com>
Co-authored-by: Ivan Vasilov <vasilov.ivan@gmail.com>
Co-authored-by: Tobias Törnros <t.tornros@gmail.com>

kangmingtay committed:
Merge branch 'master' into feat/x-d4

hf commented on apps/www/_blog/2023-12-14-supabase-auth-identity-linking-hooks.mdx:
```diff
@@ -0,0 +1,225 @@
+
+## Conclusion
+
+Thanks for reading till the end! We hope you enjoyed the Supabase Auth updates for Launch Week X: identity linking, session control, leaked password protection, and auth hooks with Postgres functions.
```
```suggestion
Thanks for reading till the end! We hope you enjoyed the Supabase Auth updates for Launch Week X: Identity Linking, Session Control, Leaked Password Protection, and Auth Hooks with Postgres functions.
```

kangmingtay committed:
Update apps/www/_blog/2023-12-14-supabase-auth-identity-linking-hooks.mdx

Co-authored-by: Stojan Dimitrovski <sdimitrovski@gmail.com>

fsansalvadore committed:
update d4 link

kangmingtay committed:
fix prettier + more updates

kangmingtay committed:
Merge branch 'feat/x-d4' of https://github.com/supabase/supabase into feat/x-d4

fsansalvadore committed:
update announcement banners

kangmingtay committed:
fix prettier

kangmingtay committed:
Merge branch 'feat/x-d4' of https://github.com/supabase/supabase into feat/x-d4

fsansalvadore committed:
X blog digest (#19687)

* Caption fixes for LWXd3 (#19680)

* Update 2023-12-13-supabase-branching.mdx

* Update 2023-12-13-supabase-branching.mdx

* lwx blog digest

* fix: add requirement on Docker client for edge funcs deploy (#19624)

* fix: add requirement on Docker client for edge funcs deploy

* fix: docker name

Co-authored-by: Han Qiao <sweatybridge@gmail.com>

* Update apps/docs/pages/guides/functions/deploy.mdx

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

* fix: prettier

---------

Co-authored-by: Han Qiao <sweatybridge@gmail.com>
Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

* Add YouTube video, form link, more internal linking. (#19685)

* Add YouTube video, form link, more internal linking.

* More internal linking.

* Edit admonition.

* Swap OG image, add planetpg tag

* Prettier

* Update name

---------

Co-authored-by: Terry Sutton <saltcod@gmail.com>

* fix: update basic auth settings form (#19708)

* fix: allow formfield to not have an icon

* fix: convert radio buttons to formfields

* fix: update copy & icons in auth settings page (#19710)

* fix: add hcaptcha and cloudflare icons

* fix: update copy

* Chore/deprecate publications store (#19655)

* Hide RLS assistant features for orgs with HIPAA

* Replace all usage of meta.publications with react query

* Deprecate publications store from meta store mobx, shift updateTableRealtime logic to SidePanelEditor

* Address comments

* Fix create trigger (#19711)

* Chore/deprecate triggers store (#19712)

* Replace all usage of meta.triggers with react query

* Deprecate and remove ttriggers mobx store

* Fix

* Update apps/studio/components/interfaces/Database/Triggers/TriggersList/TriggersList.tsx

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

---------

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

* Chore/branching update requirements (#19670)

* Update requirements to enable branching

* Remove panel query when closing subscripton plan panel

* Update apps/studio/components/layouts/AppLayout/EnableBranchingButton/BranchingPlanNotice.tsx

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

* Lint

* Lint

* Update

---------

Co-authored-by: Alaister Young <alaister@users.noreply.github.com>

* Update branching info (#19715)

* Update branching info

* swap

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

* fix: time until hackathon end count down clock (#19716)

* build-d4 (#19709)

* build-d4

* Add images and more links.

* Update description.

* ADd mdx to filename.

* Fix and update images

* Edit filename.

* replace notion links and prettify doc

* fix title image link

* update title

* Update apps/www/_blog/2023-12-14-supabase-wrappers-v02-now-available.mdx

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* Update apps/www/_blog/2023-12-14-supabase-wrappers-v02-now-available.mdx

Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* update filename,

---------

Co-authored-by: Bo Lu <lv.patrick@gmail.com>
Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>

* docs: auth passwords page (#19490)

* docs: auth passwords page

* Add redirect from old url

* apply suggestion from @charislam

Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>

---------

Co-authored-by: Kamil Ogórek <kamil.ogorek@gmail.com>
Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>
Co-authored-by: Kang Ming <kang.ming1996@gmail.com>

* Chore/deprecate functions store (#19714)

* Shift all usage of database functions from pg meta store to RQ

* Remove functions store from pg meta mobx store

* fix

* Fix

* Fixes

* fix

* Address comments

* Add build d4 (#19721)

add build item

* update digest layout

* add community meetups link

* move main stage label

---------

Co-authored-by: Jonathan Summers-Muir <MildTomato@users.noreply.github.com>
Co-authored-by: Lakshan Perera <lakshan@supabase.io>
Co-authored-by: Han Qiao <sweatybridge@gmail.com>
Co-authored-by: Charis <26616127+charislam@users.noreply.github.com>
Co-authored-by: Ramiro Nuñez Dosio <ramiro@supabase.io>
Co-authored-by: Terry Sutton <saltcod@gmail.com>
Co-authored-by: Kang Ming <kang.ming1996@gmail.com>
Co-authored-by: Joshen Lim <joshenlimek@gmail.com>
Co-authored-by: Alaister Young <alaister@users.noreply.github.com>
Co-authored-by: Tyler <18113850+dshukertjr@users.noreply.github.com>
Co-authored-by: Bo Lu <lv.patrick@gmail.com>
Co-authored-by: Copple <10214025+kiwicopple@users.noreply.github.com>
Co-authored-by: Stojan Dimitrovski <sdimitrovski@gmail.com>
Co-authored-by: Kamil Ogórek <kamil.ogorek@gmail.com>

Commit:
Update images.

Commit:
Add images.

fsansalvadore committed:
linnk lwx

kangmingtay committed:
Update apps/www/_blog/2023-12-14-supabase-auth-identity-linking-hooks.mdx

Co-authored-by: Stojan Dimitrovski <sdimitrovski@gmail.com>
